.github/ISSUE_TEMPLATE/1_Bug_report.md

```diff
@@ -34,4 +34,4 @@ WARNING: obfuscate all sensible data (IP, passwords, domain name...) -->
  - QGIS Project version:
  - OS (Windows, Linux, MacOS, Android…): 
  - Browser (Firefox, Chrome…):
- - Lizmap data stored into Sqlite (default)? PostgreSQL? MySQL? 
+ - Lizmap data stored into SQLite (default)? PostgreSQL? MySQL? 
```

Comment:
Can this be more explicit ? Which lizmap data is concerned ? Auth, logs, geo-data ?